File: kotlin-ide/j2k/new/testData/newJ2k/nullability/nullableInitializer3.java
import org.jetbrains.annotations.Nullable;

public class Test {
    public String notNullString(int p) {
        return "response";
    }

    private String notNullInitializerField = notNullString(3);
    public String notNullInitializerPublicField = notNullString(3);

    public void testProperty() {
        notNullInitializerField.charAt(0);
        notNullInitializerPublicField.charAt(0);
    }

    public void testLocalVariable() {
        String notNullInitializerVal = notNullString(3);
        notNullInitializerVal.charAt(0);
    }
}

File: j2k/testData/fileOrElement/nullability/nullableInitializer3.kt
class Test {
    fun notNullString(p: Int): String {
        return "response"
    }

    private val notNullInitializerField = notNullString(3)
    var notNullInitializerPublicField = notNullString(3)

    fun testProperty() {
        notNullInitializerField[0]
        notNullInitializerPublicField[0]
    }

    fun testLocalVariable() {
        val notNullInitializerVal = notNullString(3)
        notNullInitializerVal[0]
    }
}